src/components/PeopleFilters.tsx

```diff
@@ -1,23 +1,115 @@
-export const PeopleFilters = () => {
+import classNames from 'classnames';
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+type Props = {
+  sex: string;
+  centuries: string[];
+};
+
+export const PeopleFilters: React.FC<Props> = ({ sex, centuries }) => {
+  const [searchParam, setSearchParam] = useSearchParams();
+  const defaultNameValue = searchParam.get('name') || '';
+
+  const sexActive = ({ isActive, type }: { isActive: string; type: string }) =>
+    classNames({
+      'is-active': isActive === type,
```

Comment:
The `sexActive` function should use a boolean for `isActive` instead of a string. Consider changing the `isActive` parameter to a boolean to reflect its purpose more accurately.